src/encoded/schemas/user.json

```diff
@@ -93,9 +93,8 @@
         }
     },
     "boost_values": {
-        "title": 1.0,
-        "lab": 1.0
+        "title": 1.0
```

Comment:
master has both "title" and "lab," and when I reverted my master revert, I wasn’t sure which way the ES5 branch should go, but I had a back-end crash with "lab" in the ES5 branch. @bek felt "lab" probably doesn’t do anything even in master because it’s an object and not a value, and "boost_values" is deprecated anyway, so "lab" shouldn’t be here. I don’t know if that means eventually all "boost_values" properties should be removed from all schemas.

Reply:
boost literally has been deprecated in Elasticsearch (2.x even) but I think we still read this property out of the schemas, it's the set of default fields searched (relevant for text box mostly).   We may, in the future, default this to "all fields".